### ISSUE TITLE
[Sybil Report]

### ISSUE THREAD

**Issue by Tiodameiota1** [LayerZero-Labs/sybil-report#1500]
# Reported Addresses

0xc8c5014c7d9b9f0347d929ee7107f1f9ac2079f3
0x333c07decf3ecb8d6e84f1cd98be36c2717d76cf
0x6738dff105bd2661a67cc02b751818a3646741a4
0xe59092351b92b9f6d1df2a9d5b587a289afb4619
0xaa7cd4fe83300f9f43168719cb071512ef857de4
0x8b7afcb3a4e25910a1f113a3baa75643078dd4a8
0x06afff11f980cd6f83bf233335cdd5d6d1052738
0x93a2f0ebcf45912bde291c999ea8ef8d4bde0111
0x5eac6e9be4008448b61e708367085cff29bee969
0x81cd1d294a32b1ba344e5e2dcc6cedf1744d8018
0xa534f3e5adde6f8a00a51ee84a7fa5b291a5a816
0xfef2afa51c12cdca5255fa144a7f19f3a0527a7b
0x649f34418985d1201af5ec1b17b6f28ddc811368
0xdb686a9b77bc516a5fce6307a6ca58f6cbde1e3a
0x8c6b30b6c7db0981c773330b84d8a32fc7f35398
0xb97369c76a8e51bb5f25d42dbdf97715101ac6d0
0xbd57cdc665798435b233c6c0c0fddd6b1783d8ad
0x3475cee3b850516b68973ec686845aee7906fdf1
0xfef2afa51c12cdca5255fa144a7f19f3a0527a7b
0xaa7cd4fe83300f9f43168719cb071512ef857de4
0xa4fd31effe842bafbdd9c6dd722d9b5c7e571f8a
0xee53f3e81a6d52d467dfe1e91e3ac5b9299de109
0xdecefe0b374e304cb7b9b8f6337b9655702bc93a
0x94d2c7c7c2e531d9086ef89389f745d26e433db2
0xc9ec3de26c8011bb5d65454bb0c72e418f11f0da
0x5893999ba6c880b3538e9570e9a0f4ff6c5c05fd
0x2c43a9e45bb73842064628d516e11f03311baaff
0x058ce8b5581d2ac76babcf613ce2f1ebd6b8c2d7
0xfa55541d94d07346d91e633aa601427b91b67766
0x97dc25236aa90fda8abb60f648112f76ca080bae

# Description

almost all transactions are the same and on the same days, similar volume (different by a few dollars), all the same networks are used on the same days (it differs by 1 or 2 days to make it look like a real user), all wallets have dozens of transactions in the same day to generate volume and transactions, contracts almost all identical...

In short, they all have the same transactions and are wallets entirely for Farm, with no organic transactions on the networks other than bridge using L0.

# Detailed Methodology & Walkthrough

I used Dune panels, where I checked the number of TX, volume, active days and networks used. Then I analyzed the transactions in the Scan of each network and in the Layerzero scan, where I found similarities in all transactions between these wallets.

# Reward Address (If Eligible)

0xbAf7e2a65F4694655447fB0D32838e7D142e4a34
